Examples/ArcGISToolkitExamples/ARExample.swift

```diff
@@ -18,14 +18,63 @@ import ArcGIS
 
 class ARExample: UIViewController {
     
-    let arView = ArcGISARView(renderVideoFeed: true, tryUsingARKit: true)
+    typealias sceneInitFunction = () -> AGSScene
+    
+    /// The scene creation functions plus labels.  The functions create a new scene and perform any necessary `ArcGISARView` initialization.  This allows for changing the scene and AR "mode" (table top or full-scale).
+    private var sceneInfo: [(sceneFunction: sceneInitFunction, label: String)] = []
+    
+    /// The current scene info.
+    private var currentSceneInfo: (sceneFunction: sceneInitFunction, label: String)? {
+        didSet {
+            guard let label = currentSceneInfo?.label else { return }
+            statusViewController?.currentScene = label
+        }
+    }
+    
+    /// The `ArcGISARView` that displays the camera feed and handles ARKit functionality.
+    private let arView = ArcGISARView(renderVideoFeed: true, tryUsingARKit: true)
+    
+    /// Denotes whether we've performed a hit test yet.
+    private var didHitTest: Bool = false
+
+    // View controller displaying current status of `ARExample`.
+    private let statusViewController: ARStatusViewController? = {
+        let storyBoard = UIStoryboard(name: "ARStatusViewController", bundle: nil)
```

Comment:
When a storyboard contains only one view controller, I recommend using a xib instead. It is much easier to instantiate (`ARStatusViewController()`, assuming the xib shares the same name as the view controller) and makes it clear that the view controller isn't part of a larger workflow. Yes, static table view cells are not supported in a xib, but I would recommend making the the table view's data source more robust anyway. Right now, if another scene were to be added, that would require a change to this view controller, `ARStatusViewController`, and the storyboard. Ideally only this view controller or `ARStatusViewController` would need to change, depending on who is responsible for populating the backing source used by the table view's data source.

Reply:
I made the change to a .xib.